### PR TITLE
LOG-660: Added fluent-plugin-label-router to support Forwarder Label Selector

### PR DESCRIPTION
**Pull request by k-keiichi-rh**
### Description
This PR vendors [the label router plugin](https://github.com/banzaicloud/fluent-plugin-label-router) into the fluentd image.

/cc @alanconway @jcantrill
/assign

### Links
- Github issue: https://github.com/openshift/cluster-logging-operator/issues/787
- JIRA: https://issues.redhat.com/browse/LOG-660
- Enhancement proposal: https://github.com/openshift/enhancements/blob/master/enhancements/cluster-logging/forwarder-label-selector.md

### CODE REVIEW

File: fluentd/Gemfile.lock [hunk 1]
```diff
@@ -59,6 +59,8 @@ GEM
       fluentd (>= 0.10.58, < 2)
```
Comment (alanconway) [inline fluentd/Gemfile.lock:59]: I think this should be replaced by the fluentd requirement below.
```suggestion
      fluentd (>= 0.14.10, < 2)      
```

File: fluentd/Gemfile.lock [hunk 2]
```diff
@@ -59,6 +59,8 @@ GEM
...
       ltsv
       ruby-kafka (>= 0.7.8, < 2)
+    fluent-plugin-label-router (0.2.4)
+      fluentd (>= 0.14.10, < 2)
```
Comment (alanconway) [inline fluentd/Gemfile.lock:63]:
```suggestion
```

**Review by alanconway — CHANGES_REQUESTED** [review]
One minor change, otherwise looks good.

File: fluentd/Gemfile.lock (hunk 1 above)
Comment (k-keiichi-rh) [reply fluentd/Gemfile.lock:59]: Thanks. I will fix it. 
Just confirmation. fluent-plugin-label-router requires ">= 0.14.10". So minimum version for other components should be also raised. Is my understanding correct? 
If yes, do I need to raise the lower version of other components(0.14.0 and 0.14.2)?
Or is it no problem if 0.14.x is same for the compatibility of fluentd?

Comment (k-keiichi-rh) [reply fluentd/Gemfile.lock]: I updated my patch. If there is no problem, please merge it.

**Pull request merged**